camel-djl/README.md

```diff
@@ -0,0 +1,95 @@
+# DJL Component in Apache Camel
+
+An example application that demonstrates simple HTTP-service to classify images using Zoo Model.
+
+Code example:
+
+```java
```

Comment:
We probably shouldn't put a copy of the code in the readme or it will end up diverging if the example is updated. Maybe just link to the code demo?

Reply:
Fixed.